File: bot.js
const TelegramBot = require('node-telegram-bot-api');
const token = '7225739052:AAF0e4cNK9WM0EZI7YnzK_S5t0tvLIOtMsQ';
const bot = new TelegramBot(token, { polling: true });

bot.onText(/\/start/, (msg) => {
    const chatId = msg.chat.id;
    const opts = {
        reply_markup: {
            inline_keyboard: [
                [
                    {
                        text: 'افتح التطبيق',
                        web_app: { url: 'https://its-mosab.github.io/Bbb/index.html' }
                    }
                ]
            ]
        }
    };
    bot.sendMessage(chatId, 'اضغط على الزر لفتح التطبيق:', opts);
});

File: bot.py
from telegram import InlineKeyboardButton, InlineKeyboardMarkup, Update, WebAppInfo
from telegram.ext import Updater, CommandHandler, CallbackContext

def start(update: Update, context: CallbackContext) -> None:
    keyboard = [[InlineKeyboardButton("افتح التطبيق", web_app=WebAppInfo(url="https://its-mosab.github.io/Bbb/index.html"))]]
    reply_markup = InlineKeyboardMarkup(keyboard)
    update.message.reply_text('اضغط على الزر لفتح التطبيق:', reply_markup=reply_markup)

def main() -> None:
    updater = Updater("7225739052:AAF0e4cNK9WM0EZI7YnzK_S5t0tvLIOtMsQ")

    dispatcher = updater.dispatcher
    dispatcher.add_handler(CommandHandler("start", start))

    updater.start_polling()
    updater.idle()

if __name__ == '__main__':
    main()
